game-server/data/handlers/instance/pvp/KamarBattlefieldInstance.java

```diff
@@ -256,6 +256,7 @@ public void handleUseItemFinish(Player player, Npc npc) {
 				updatePoints(player, player.getRace(), npc.getObjectTemplate().getL10n(), 200);
 				if (player.getRace() == Race.ELYOS) {
 					spawn(701900, npc.getX(), npc.getY(), npc.getZ(), npc.getHeading());
+					npc.getController().delete();
 				} else {
 					spawn(701901, npc.getX(), npc.getY(), npc.getZ(), npc.getHeading());
 					npc.getController().delete();
```

Comment:
```suggestion
				spawn(player.getRace() == Race.ELYOS ? 701900 : 701901, npc.getX(), npc.getY(), npc.getZ(), npc.getHeading());
				npc.getController().delete();
```

Reply:
GitHub won't let me remove the closing brace below the changed lines. Please remove it manually after applying this change.